fluidreview/management/commands/importfluidusers.py

```diff
@@ -0,0 +1,24 @@
+"""
+Command for importing FluidReview users
+"""
+from django.core.management import BaseCommand
+
+from fluidreview import serializers
+from fluidreview.api import process_user, list_users
+
+
+class Command(BaseCommand):
+    """
+    Create user and profile objects for each FluidReview user
+    """
+    def handle(self, *args, **options):
+        """
+        Run the command
+        """
+        for fluid_user in list_users():
+            serializer = serializers.UserSerializer(data=fluid_user)
+            if serializer.is_valid(raise_exception=False):
+                process_user(serializer.data)
+                self.stdout.write('Processed User & Profile for {}'.format(fluid_user['email']))
+            else:
+                self.stdout.write('Invalid data for {}: {}'.format(fluid_user['email'], serializer.errors))
```

Comment:
Do you know if invalid data is a common or rare case? Can you keep track of if there are invalid cases and return a non-zero exit code in that case?

Reply:
I haven't yet encountered it in testing with the API, so hopefully it is rare.  Will add a non-zero exit code if it happens.